detail about API token expiration/lifetime (#611)
neo4j/docs-aura: modules/ROOT/pages/platform/api/authentication.adoc

=== Token expiration

> Tokens are temporary access tokens used for authentication when making API requests. 
> A token expires after one hour. 

Reviewer: ```suggestion
Access tokens are temporary and expire after one hour.
```
It's already established that they are used for authentication when making API requests.

=== Token expiration

> Tokens are temporary access tokens used for authentication when making API requests. 

Reviewer 2: This should be moved to where the tokens are first introduced (I think in the "Authenticating API requests" section).

> They expire after one hour. 
> If you send a request to the Aura API using an expired token, you will receive a 403 Forbidden response. 

Reviewer 2: ```suggestion
A token expires after one hour. 
If you send a request to the Aura API using an expired token, you will receive a `403 Forbidden` response. 
```

> To continue using the API, you must obtain a new token.
> 
> In the *Account Details* section of the Aura console, Aura API credentials refer to OAuth clients (Client ID and Client Secret). 
> These credentials do not expire unless deleted and are used to generate access tokens, which do expire.

Reviewer 2: ```suggestion
To continue using the API, you must obtain a new token using the Aura API credentials.
```

Reviewer 2: I think this is enough, given that the API credentials and the token creation are described just in the previous section.

Reviewer: +1

When an access token is created using a Client ID and Client Secret, the response payload specifies its expiration time in seconds. 
Once the token expires, the application must request a new one.

> 

Reviewer 3: From a technical point of view there is nothing incorrect with this description. 
I have no input on wording. I will however leave this link here https://datatracker.ietf.org/doc/html/rfc6749#section-1.1 which defines the term 'client' in case it is useful as a hyperlink or for any other reason.

> 

Reviewer 2: I think this whole paragraph is redundant, since this is mentioned in the previous section. Maybe add a description of the `expires_in` field in the "Response body example" section after the description of `access_token`?

> 
> 
> 
> 

Reviewer 2: ```suggestion
```
